app/controllers/analytics/districts_controller.rb

```diff
@@ -70,7 +70,7 @@ def set_dashboard_analytics(period, prev_periods)
       analytics_cache_key_dashboard(period),
       @organization_district.dashboard_analytics(period: period,
                                                  prev_periods: prev_periods,
-                                                 include_current_period: @show_current_month))
+                                                 include_current_period: @show_current_period))
```

Comment:
Can we inline this?

Reply:
`show_current_period` needs to be declared, it's being used in the view (`_analytics_table.html.erb`)